### PR TITLE
Including opaque_keys.edx.tests in exported packages

### PR DESCRIPTION
**Pull request by clintonb**
This package is needed by ccx-keys.

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @feanil @nedbat @benpatterson

File: setup.py [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 setup(
     name="edx-opaque-keys",
-    version="0.2.0",
+    version="0.2.1",
     author="edX",
     url="https://github.com/edx/opaque-keys",
     packages=[
         "opaque_keys",
         "opaque_keys.edx",
+        "opaque_keys.edx.tests",
```
Comment (nedbat) [inline setup.py]: Perhaps it would be safest now to include everything.  "opaque_keys.tests" is still missing.

Comment (clintonb) [inline setup.py]: Good point. Will do.

**Comment by clintonb** [comment]
@nedbat updated

**Pull request merged**